CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to the Imperative package will be documented in this file.
 
 ## Recent Changes
 
+- Remove temporary build modifications that were used to enable zowe-cli to build with an unpublished imperative.
```

Comment:
Should this PR update the changelog, since these aren't user-facing changes? We can add the "no-changelog" label to this PR to make the build pass without a changelog update.

Reply:
Done.